libbeat/ml-importer/importer.go

```diff
@@ -228,32 +228,26 @@ func checkResponse(r []byte) error {
 	}
 	for _, dashboard := range resp.Kibana.Dashboard {
 		if !dashboard.Success {
-			if dashboard.Exists {
+			if dashboard.Exists || strings.Contains(dashboard.Error.Message, "version conflict, document already exists") {
 				logp.Debug("machine-learning", "Dashboard already exists: %s", dashboard.ID)
-			} else if strings.Contains(dashboard.Error.Message, "version conflict, document already exists") {
-				continue
 			} else {
 				errs = append(errs, errors.Errorf("error while setting up dashboard: %s", dashboard.ID))
 			}
 		}
 	}
 	for _, search := range resp.Kibana.Search {
 		if !search.Success {
-			if search.Exists {
+			if search.Exists || strings.Contains(search.Error.Message, "version conflict, document already exists") {
```

Comment:
Was wondering here and below the same as commented above.

Reply:
Done